person_capture/gui_app.py

```
@@ -5389,14 +5462,320 @@ def _resolve_ffmpeg_bin(self) -> Optional[str]:
+        nits = float(getattr(self.cfg, "sdr_nits", 125.0))
+        pref = str(getattr(self.cfg, "hdr_tonemap_pref", "auto") or "auto").lower()
+        quality = str(getattr(self.cfg, "hdr_sdr_quality", "resolve_like") or "resolve_like").lower()
+        algo = str(getattr(self.cfg, "hdr_sdr_tonemap", "spline") or "spline").lower()
+        if algo not in {"spline", "bt.2390", "st2094-40", "mobius", "hable", "reinhard", "clip"}:
+            algo = "spline"
+        gamut = str(getattr(self.cfg, "hdr_sdr_gamut_mapping", "perceptual") or "perceptual").lower()
+        if gamut not in {"perceptual", "relative", "saturation", "clip"}:
+            gamut = "perceptual"
+        try:
+            contrast_recovery = max(0.0, min(2.0, float(getattr(self.cfg, "hdr_sdr_contrast_recovery", 0.30))))
+        except Exception:
```

Ruff (0.15.11)

[warning] Do not catch blind exception: `Exception`

(BLE001)